nixos/modules/services/web-servers/unit/default.nix

```diff
@@ -94,6 +94,7 @@ in {
       path = with pkgs; [ curl ];
       preStart = ''
         test -f '/run/unit/control.unit.sock' || rm -f '/run/unit/control.unit.sock'
+        test -f '${cfg.stateDir}/conf.json'   || rm -f '${cfg.stateDir}/conf.json'
```

Comment:
This looks dangerous. What's written into `conf.json`? Why does it end in `${cfg.stateDir}` in first place, if we don't want it?

Reply:
In `conf.json` last working configuration is stored.
At the `postStart` stage, the current configuration is loaded.